func/sgx/chaining_sgx_wamr.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
```

Comment:
I moved these to a folder called `sgx` as we want to make a distinction between WAMR with and without SGX